easier_net/sier_net.py

```diff
@@ -298,6 +299,7 @@ def score(self, x: np.ndarray, y: np.ndarray) -> float:
         return -self.score_criterion(self.net(torch.Tensor(x)), torch_y).item()
 
     def predict(self, x: np.ndarray) -> np.ndarray:
+        self.n_inputs = x.shape[1]
```

Comment:
hm, this line shouldn't be here. it should only show up in `fit`... we don't modify the model after fitting